notebooks/122-pytorch-post_training_quantization_and_sparsity_pot/122-pytorch-post-training-quantization-sparsity-pot-public.ipynb

```diff
@@ -0,0 +1,668 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # On Windows, this script adds the directory that contains cl.exe to the PATH to enable PyTorch to find the
</pre><p>does it really required? as I know this part of code in previous notebooks used for NNCF quantization using PyTorch to be able load custom pytorch layers realization, it is not used in your notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>